SketchPage.qml

```diff
@@ -16,6 +16,12 @@ StandardPage {
             console.log("onCurrentStrokeWidthChanged!" + penModel.currentStrokeWidth + (penModel.currentStrokeWidth.toFixed(1) == 2.5))
             featuresId.checked = windowId.fuzzyEquals(penModel.currentStrokeWidth, 2.5);
         }
+        onUndoStackChanged: {
```

Comment:
The pen model should just have a property to the qundostack, the qml code can just use the qundostack properties directly.

Reply:
I did that at one point, but I couldn't figure out how to have the UI connect to a signal when the undo stack changed (so that the buttons could be disabled when inapplicable), so I added these wrappers. Is there a way to do that? If so, it wasn't obvious to me. (Never used Qt before now and it's a bigger ecosystem than I thought.)